apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/RpcRetryConsumerTest.cs

```diff
@@ -210,11 +208,11 @@ private void SetupTestReceiveThrows(Action<FakeSequentialThreadingTimer, Mock<IC
         /// </summary>
         private class FakeSequentialThreadingTimer : ISequentialThreadingTimer
         {
-            private Action _callback;
+            internal Action Callback = null;
```

Comment:
This can't be readonly due to the api of ISequentialThreadingTimer (which I need to revisit)